spk/octoprint/Makefile

```diff
@@ -44,9 +44,9 @@ octoprint_extra_install: $(STAGING_DIR)/share/OctoPrint
 	install -m 755 -d $(STAGING_DIR)/app/images
 	for size in 16 24 32 48 72; do \
 		convert $(SPK_ICON) -thumbnail $${size}x$${size} \
-		        $(STAGING_DIR)/app/images/$(SPK_NAME)-$${size}.png ; \
+			$(STAGING_DIR)/app/images/$(SPK_NAME)-$${size}.png ; \
 	done
 
 $(STAGING_DIR)/share/OctoPrint:
 	install -m 755 -d $(STAGING_DIR)/share
-	cd $(STAGING_DIR)/share && git clone https://github.com/foosel/OctoPrint.git OctoPrint
+	cd $(STAGING_DIR)/share && git clone -b devel https://github.com/foosel/OctoPrint.git OctoPrint
```

Comment:
See my earlier comment. I'd rather see the package using the master branch, and it would be even better to stick with actual releases (https://github.com/foosel/OctoPrint/releases).
At the very least, you need to use a specific commit to checkout.